src/AddTodoForm.jsx

```diff
@@ -1,8 +1,15 @@
-function AddTodoForm() {
+function AddTodoForm(todoprops) {
```

Comment:
for consistency React developers using word 'props' or refactor props (we'll learn it soon).

Reply:
Understood, thanks.